core/dbt/task/debug.py

```diff
@@ -58,6 +62,23 @@
 FILE_NOT_FOUND = "file not found"
 
 
+SubtaskStatus = namedtuple(
+    "SubtaskStatus", ["log_msg", "run_status", "details", "summary_message"]
+)
+
+
+class RunStatus(Enum):
+    PASS = 1
+    FAIL = 2
+    SKIP = 3
+
+
+DEBUG_RUN_STATUS: Dict[str, bool] = {
```

Comment:
I'd really rather not have fandom boolean literals if possible because of loss of context, even if it seems obvious. This is just syntax sugar but it is more descriptive than having people calculate the boolean on the fly or think about how True/False corresponds to the return value.

Reply:
Using a bool value enum now